plugins/apps/eiskaltdcpp.mk

```diff
@@ -0,0 +1,80 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := eiskaltdcpp
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 4f5ebdea7a53f5d2492f391add9c1848ba377919
+$(PKG)_CHECKSUM := ebada8be7c280e3433d69f132e6a46d47da0fe31a1b201fe397b8b18b6304e73
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://github.com/$(PKG)/$(PKG)/archive/$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc aspell boost jsoncpp libidn lua miniupnpc nsis qt5
```

Comment:
I think full `qt5` target requires too much build time and disk space.
Could you specify only really used `qt` modules?